src/GraphQL/General/UploadType.php

```diff
@@ -25,15 +25,9 @@
 
 class UploadType extends ScalarType
 {
-    /**
-     * @var string
-     */
-    public $name = 'Upload';
+    public string $name = 'Upload';
 
-    /**
-     * @var string
-     */
-    public $description =
+    public null | string $description =
```

Comment:
```suggestion
    public string $description =
```

Just a minor thing, but there is no reason for the description to allow null values, right?

Reply:
It has to be defined `null | string` because it extends the base type class, PHPStan will throw an error when you don't

Reply:
```suggestion
    public ?string $description =
```
Aaah you are right, of course. Sorry, I missed that. Maybe we can change it to `?string` though, since this is the style we want to use from Pimcore 11 on.